rocAL/rocAL/include/loaders/image/cifar10_data_loader.h

```diff
@@ -40,7 +40,7 @@ class CIFAR10DataLoader : public LoaderModule
     void reset() override;
     void start_loading() override;
     std::vector<std::string> get_id() override;
-    decoded_image_info get_decode_image_info() override;
+    decoded_sample_info get_decode_sample_info() override;
```

Comment:
Please check if it is better to add THROW Statements here instead of using override